wifi/scheme.py

```diff
@@ -158,6 +158,12 @@ def activate(self):
 
         return self.parse_ifup_output(ifup_output)
 
+    def deactivate(self):
+        """
+        Disconnect from the network without bringing down the interface
+        """
+        return subprocess.call(['/sbin/dhclient', '-r', self.interface], stderr=subprocess.STDOUT)
```

Comment:
The return value of whatever program is called.  In the case of dhclient that would be 0 on success and non-zero on failure.